### PR TITLE
MAPS: It's for breakfast today

### PR DESCRIPTION
**Pull request by jwiggins**
Wormy steps...
Screenshot 2023-02-04 at 10 59 40

### CODE REVIEW

File: cart/player.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ func (player *Player) Draw(t int32) {
 	if t%mod == 0 {
 		player.incrementFrame()
 	}
-	player.Sprite.Draw(player.X, player.Y)
+	// Keep the player centered on screen
+	player.Sprite.Draw(120, 114)
```
Comment (fjebaker) [inline cart/player.go]: We can add a stretch to get better camera movement later!

**Review by fjebaker — COMMENTED** [review]
LGTM but is it now 8x8 movement or do we still move a pixel at a time?

File: tic80/map.go [hunk 2]
```diff
@@ -3,37 +3,39 @@ package tic80
 import "unsafe"
 
 const (
-	sHEIGHT int32 = 17
-	sWIDTH        = 30
+	SCREEN_TILE_HEIGHT  int32 = 17
```
Comment (JR-Mitchell) [inline tic80/map.go:6]: Good to see these variable names being made clear!

File: cart/world.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	X, Y int32
+}
+
+func NewWorld(player *Player) World {
+	tileX, tileY := worldPosition(player)
+	return World{tileX, tileY}
+}
+
+func worldPosition(player *Player) (int32, int32) {
+	playerTileX := (player.X / 8) - (tic80.SCREEN_TILE_WIDTH / 2)
```
Comment (JR-Mitchell) [inline cart/world.go]: We need to decide whether we want smooth player movement or block-by-block movement

Comment (jwiggins) [reply cart/world.go]: @fjebaker seems to be implementing smooth movement right now

**Review by JR-Mitchell — APPROVED** [review]
LGTM

**Pull request merged**